modules/data/context_menu.components.json

```diff
@@ -1,6 +1,7 @@
 {
-      "context": "chrome",
-      "context-menu-search-selected-text": {
+    "context": "chrome",
+    "do-not-cache": true,
```

Comment:
Does this imply that nothing in this file would be cached? And then the `doNotCache` group is to be used for individual elements?